packages/react-pose/src/components/PoseElement.tsx

```diff
@@ -76,13 +79,18 @@ export class PoseElement extends React.PureComponent<PoseElementProps> {
     }
   }
 
+  getRef(): Element {
+    return this.hostRef || this.innerRef || this.ref;
```

Comment:
Order here is important: Favouring `hostRef` (https://github.com/facebook/react/issues/11401) first, then any `innerRef` (for Styled Components support), then the original `ref`.

I'm not sure if the values could get out of date at all? Is there ever a case where the innerRef gets updated, but the hostRef doesn't? It all feels a bit icky, but less icky than using `.findDOMNode()` (which also doesn't work with Fragments).